Tasks/DeployVisualStudioTestAgent/InstallTestAgent.ps1

```diff
@@ -25,9 +25,24 @@ function InstallTestAgent2017 {
     }
 
     # First we need to install the certificates for TA 2017
-    $SetupDir = Split-Path -Path $SetupPath
-    Write-Verbose "Installing test agent certificates"
-    Get-ChildItem -Path "$SetupDir\certificates\*.p12" -ErrorAction SilentlyContinue | Import-PfxCertificate -CertStoreLocation Cert:\CurrentUser\My -Exportable
+    $SetupDir = Split-Path -Path $SetupPath    
+
+    $osVersion = [environment]::OSVersion.Version
+    $certFile = Get-ChildItem -Path "$SetupDir\certificates\*.p12" -ErrorAction SilentlyContinue
```

Comment:
>ChildItem [](start = 20, length = 9)

does this return multiple items? also i forgot but we should test with .zip as well (disonnected scenarios)